File: Python/8py 1/get_status.py
"""
|--------------------------------------------------------------------------
| Unexpected parsing
|--------------------------------------------------------------------------
|
"""

def get_status(is_busy):
   return {"status": "busy" if is_busy else "available"}


get_status = lambda b:{'status':['available','busy'][b]}


def get_status(is_busy):
   status = "busy" if is_busy else "available"
   return {"status" : status}


print(get_status(True)["status"], "busy")
print(get_status(False)["status"], "available")

File: JavaScript/8js 2/getStatus.js
/*
|--------------------------------------------------------------------------
| Unexpected parsing
|--------------------------------------------------------------------------
|
*/

function getStatus(isBusy) {
   let msg = isBusy ? "busy" : "available";
   return {
      status: msg,
   };
}


function getStatus(isBusy) {
   return { status: isBusy ? "busy" : "available" };
}


function getStatus(isBusy) {
   return { status: { true: "busy", false: "available" }[isBusy] };
}


console.log(getStatus(true).status, "busy");